classes/controllers/grid/issues/IssueGridHandler.inc.php

```diff
@@ -184,59 +183,62 @@ function uploadFile($args, $request) {
 	}
 
 	/**
-	 * Update a issue
+	 * Delete an uploaded cover image.
 	 * @param $args array
+	 *   `coverImage` string Filename of the cover image to be deleted.
+	 *   `issue` int Id of the issue this cover image is attached to
```

Comment:
eventually to call it issueId -- as always -- but not a thing if it stays as it is, I believe...